core/build.gradle

```diff
@@ -8,6 +8,7 @@ android {
         targetSdkVersion rootProject.ext.targetSdkVersion
 
         multiDexEnabled false
+        vectorDrawables.useSupportLibrary true
```

Comment:
Have you tested this change on emulators with different API versions? Does it increase the apk size by shipping vector drawables as png?

Reply:
No, but I don't think it actually changes anything in that way. I'll change it back anyway.